books/free-programming-books-langs.md

```diff
@@ -2229,6 +2229,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Just Enough Ruby to Get By](http://jasonkim.ca/projects/just_enough_ruby_to_get_by/)
 * [Learn Ruby First](https://essenceofchaos.gitbooks.io/learn-ruby-first/content/) - Frederick John
 * [Learn Ruby in Y minutes](https://learnxinyminutes.com/docs/ruby/)
+* [Learn Ruby on Rails](https://learn-rails.com/products/learn-rails-1.pdf) - Daniel Kehoe (PDF)
```

Comment:
Book already in the repository, available at: [link](https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-langs.md#ruby-on-rails)